build.gradle

```diff
@@ -1,15 +1,12 @@
-import edu.wpi.first.gradlerio.deploy.roborio.RoboRIO
-
 plugins {
```

Comment:
I'm going to assume that this file was automatically generated, but the old version crucially had the correct main class